src/api/telephony/portability/document/telephony-portability-document.v6.service.js

```diff
@@ -21,7 +21,8 @@ angular.module('ovh-api-services').service('OvhApiTelephonyPortabilityDocumentV6
     },
     getDocument: {
       method: 'GET',
-      isArray: true,
+      isArray: false,
+      interceptor,
```

Comment:
Strange to have an interceptor on a `GET`, especially as you don't cache the query. Moreover I think the `isArray` could be omitted as it is the default behaviour